components/InstallHandlerBtn.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <q-btn
+    label="Manage Credentials"
+    :loading="loading"
+    color="primary"
+    v-bind="options"
+    @click="installHandler" />
+</template>
+
+<script>
+/*!
+ * Copyright (c) 2018-2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import {installHandler} from 'web-credential-handler';
+
+export default {
+  name: 'InstallHandlerButton',
+  props: {
+    // can be used to pass options to the q-btn
+    // @see https://quasar.dev/vue-components/button
+    options: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+  data() {
+    return {
+      loading: false
+    };
+  },
+  methods: {
```

Comment:
Since this is a new component, we should use the new Vue 3 style with `setup` and composition API. If that's too much to do right now, we can file an issue to get this component and others updated to the new style instead.

Reply:
I can update it in a moment. I didn't know we wanted to use setup.

Reply:
And to be clear, I mean use the `setup()` function, not the macro stuff.

Reply:
Used `setup` here: https://github.com/digitalbazaar/bedrock-vue-wallet/pull/29/commits/b0524dc2ea59ec379cc4dfcefa8fb30c6cd96c4f

Reply:
The methods will need to be moved into setup too, etc.

Reply:
the methods were moved into the setup step, but this component was removed here: https://github.com/digitalbazaar/bedrock-vue-wallet/pull/29/commits/151e1a61046fd8feab9299c233ba961748d7e94f